python/taichi/ui/scene.py

```diff
@@ -93,8 +93,18 @@ def lines(self,
               width,
               indices=None,
               color=(0.5, 0.5, 0.5),
-              per_vertex_color=None):
+              per_vertex_color=None,
+              vertex_offset: int = 0,
```

Comment:
Is it better to give `vertex_offset` default to None as well, so that you have a way to know it's not set by users?

Reply:
Here user-no-assignment behavior means no vertex offset, which means `vertex_offset` is 0.